lib/mayaUsd/resources/ae/usd-shared-components/src/python/usdSharedComponents/collection/expressionWidget.py

```diff
@@ -45,31 +46,20 @@ def __init__(self, collection: Usd.CollectionAPI, parent: QWidget, expressionCha
         mainLayout.addWidget(self._expressionText, 1)
 
         self._expressionText.installEventFilter(self)
-        self.update()
+        self.setLayout(mainLayout)
```

Comment:
this is in general not needed - as creating the mainLayout in line 21 already did set it to the widget - as the layout is getting created with this widget as a parent. But cannot hurt, as well.

Reply:
I think it might have been due to the merge conflicts being resolve like this. I can remove the line.